testdata/scripts/lines.txt

```diff
@@ -0,0 +1,62 @@
+garble -debugdir=.obf-src build
+
+env TINY_PATTERN='^\/\/line :1$'
+env DEFAULT_PATTERN='^\/\/line \w\.go:[1-9][0-9]*$'
+env DEFAULT_STACK_PATTERN='^\t\w\.go:[1-9][0-9]*(\s\+0x[0-9a-f]+)?'
+env TINY_STACK_PATTERN='^\t\?\?:[0-9][0-9]*(\s\+0x[0-9a-f]+)?$'
+
+# Default mode
+
+# Check for file name leak protection
+grep $TINY_PATTERN $WORK/.obf-src/main/main.go

```

Comment:
Why should the normal mode contain the tiny pattern?

Reply:
Yes. I insert //line :1 at the beginning of each file to prevent any leaks in file names.

Reply:
This does not affect the default mode.